locale/po/grassmods_fr.po

```diff
@@ -29805,9 +29821,8 @@ msgid ""
 msgstr ""
 
 #: ../locale/scriptstrings/r.maxent.train_to_translate.c:61
-#, fuzzy
 msgid "Disable linear features"
-msgstr "entités simples"
+msgstr ""
```

Comment:
There was many strings that had nonsense translations, and the source text didn't correspond to a real file in the repo. Is it like a merge or rebase that went wrong at some point. Looking at the history, it was quite old too.

Reply:
That's AFAIK normal: the string was marked before as `#, fuzzy` (kind of randomly translated and marked as inactive)  and has now been emptied. I don't see a problem here.

Reply:
I know, these are all my translations from yesterday. But in the interface, there was a lot of translations that didn't match, as if it was a translation for another string, but in the 15 strings near, it didn't seem to match something

Reply:
Does the fuzzy flag match the checkbox saying needs verification (or in the French interface, "à vérifier" (to verify))?

Reply:
Note that the string comes from addons and should be there:

```shell
~/software/grass-addons [grass8|…3⚑ 1] 
15:21 $ ag 'Disable linear features'
src/raster/r.maxent.train/r.maxent.train.py
194:# % label: Disable linear features
```

Indeed, in Weblate it isn't present. Maybe just locally at your end?

Reply:
What do you mean locally? I was using the weblate

https://weblate.osgeo.org/search/grass-gis/?q=Disable+linear+features+and+language%3Afr&sort_by=-priority%2Cposition&checksum=

If we look at this string, other languages, I can at least see that the Portuguese translation had a similar translation to the french one, without knowing Portuguese
https://weblate.osgeo.org/translate/grass-gis/grassmods/fr/?checksum=a5401246b85772d7&q=Disable+linear+features+and+language%3Afr&sort_by=-priority%2Cposition#translations

It says it refers to https://github.com/OSGeo/grass/blob/main/locale/scriptstrings/r.maxent.train_to_translate.c#L61

Reply:
I think I understand the problem: when (recently) creating the POT file,  @HuidaeCho seems to have addons like r.maxent locally installed. Due to that the strings went into the POT file while not matching any PO file.

Importantly, addons must be deinstalled *before* running the procedure to update the POT/PO files.

Reply:
Are addons translations handled in weblate too?

Reply:
No, so far they are untranslated.

Reply:
If they would be added, before fixing this "bug", it would be great, as I see some amount of strings are translated (and not dating from only 3 days, some 8 months and more) and they belong to addons. It would be a shame to lose them, especially if they fill in the gaps of known concepts but still aren't applied yet in the strings.

Reply:
> I think I understand the problem: when (recently) creating the POT file, @HuidaeCho seems to have addons like r.maxent locally installed. Due to that the strings went into the POT file while not matching any PO file.
> 
> Importantly, addons must be deinstalled _before_ running the procedure to update the POT/PO files, e.g. using
> 
> `grass --tmp-project XY --exec g.extension.all -f operation=remove`

@neteler Yes, I intentionally tried to add all compilable addons before the process because I assumed that addon messages should also be translated and I thought they were back in Transifex.

Reply:
If there would be a different file, even if it syncs back to the addons repo (pointing to the correct files) it would be great.

Reply:
The entire translate chain for addons needs to be set up, the addon pot/po files created etc. Then we could attempt to transfer the addon-related translations from main (where they should not be) to addons. Volunteer(s) needed...